browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -579,6 +579,9 @@ month...
       <message name="IDS_BRAVE_UI_BACKUP_WALLET_DESC" desc="A notification message description about backing up the wallet">
         Please backup your Brave wallet.
       </message>
+      <message name="IDS_BRAVE_WALLET_PORTFOLIO_BALANCE_UPDATING">
```

Comment:
add `translateable="false"` argument as we don't want to translate it

Reply:
thanks, fixed with https://github.com/brave/brave-core/pull/10174/commits/2c83ac338c7c317997db171e4bd4ad21b79bcc1f